src/pages/trainer/training/edit/components/editor.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import {ToolbarContainerComponent} from './toolbar';
+const classNames: any = require('./editorStyles.scss');
+
+interface IProps {
+  content: string;
+  cursorStartPosition: number;
+  shouldSetEditorFocus: boolean;
+  className: string;
```

Comment:
Yeah, but I think that the matter is that is exactly call as the react property className, may be could be fit better, something like componentNameClassStyles, or just componentNameClass.